guides/source/rails_on_rack.md

```diff
@@ -217,6 +219,10 @@ config.middleware.delete! ActionDispatch::Executor
 
 Much of Action Controller's functionality is implemented as Middlewares. The following list explains the purpose of each of them:
 
+**`ActionDispatch::HostAuthorization`**
+
+* Guards from DNS rebinding attacks by explicitly permitting the hosts a request can be sent to, and is passed the options set in `config.host_authorization`.
```

Comment:
I realize this is [exactly what the API docs say](https://github.com/rails/rails/blob/8b2511b8ff28bc991ec459d798b8296195b423a3/actionpack/lib/action_dispatch/middleware/host_authorization.rb#L4-L6), but I feel like we can improve it a little for this guide:

```suggestion
* Guards from DNS rebinding attacks by explicitly permitting the hosts a request can be sent to. See the [configuration guide](configuring.html#actiondispatch-hostauthorization) for configuration instructions.
```

Reply:
Done